### ISSUE TITLE
PSK Security Scheme needs work

### ISSUE THREAD

**Issue by mmccool** [w3c/wot-thing-description#178]
I noticed that the "psk" scheme includes a parameter "psk" which is defined as the "pre-shared key".   I assume this is supposed to the pre-shared *public* key.  To be clear private (secret) information should not be in the TD.  A PSK needs to be "pre-shared" out of band, eg during provisioning; that is the entire point.  Can we clarify?  Perhaps the "psk" value should be called something else, like "cert", if it is a certificate containing a public key that can be used to confirm the given identity via digital signing against the related private key contained in the Thing (for example).

Also the ontology defines the "psk" value twice... anyway, this scheme needs some work.

**Comment by mkovatsc** [comment]
PSKs are symmetric -- "Pairwise Symmetric Keys" ;) -- so there is no public one. Also, the PSK of course must not go into the TD.

@vcharpenay please fix the ontology

**Comment by mkovatsc** [comment]
@vcharpenay is working on it.

**Comment by vcharpenay** [comment]
See #180

**Comment by mmccool** [comment]
I will take a look; just deleting the "psk" field in the "psk" scheme (and its duplicate...) is the right resolution, I think.

**Comment by mmccool** [comment]
When doing [PR183](https://github.com/w3c/wot-thing-description/pull/183) I noted the "psk" fields still existed in the ontology... so I deleted them.   The updated figure also does not include this field.

**Comment by mmccool** [comment]
Oops, I see that the psk fields WERE deleted in VC's PR.   Well, my edits should not cause a conflict.   Also, I have another PR out that renames pname to name... I see Victor's PR also does that.   Again, there should not be a conflict with my pname->name PR, although I fixed a few other things related to this outside the ontology.

Sigh... hopefully Sebastian can sort it out.   It would have been nicer to do these edits sequentially and not in parallel.   I suggest merging PR180 first, then try PR183, then try to merge PR177.  If there are conflicts in my two PRs let me know and I will fix them.

**Comment by mmccool** [comment]
Never mind, I reviewed PR177 and there were not any changes that PR180 does not include, so I closed PR177.   There should not be a conflict between 180 and 183 but if git complains it should be an easy thing to fix.

**Comment by mkovatsc** [comment]
Confirmed in call on 8 Aug 208. Closing.
